### PR TITLE
upgrade DB to 5.19.0

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Update DB to 5.19

/cc @amyblais

### CODE REVIEW

File: store/sqlstore/upgrade.go [hunk 1]
```diff
@@ -18,7 +18,8 @@ import (
 )
 
 const (
-	CURRENT_SCHEMA_VERSION   = VERSION_5_18_0
+	CURRENT_SCHEMA_VERSION   = VERSION_5_19_0
+	VERSION_5_18_0           = "5.19.0"
```
Comment (jespino) [inline store/sqlstore/upgrade.go]: typo here `s/8/9/`

Comment (cpanato) [reply store/sqlstore/upgrade.go]: i know :) thanks!

Comment (cpanato) [reply store/sqlstore/upgrade.go]: fixed

**Comment by cpanato** [comment]
ping @lindalumitchell

**Pull request merged**